src/main/cpp/Robot.cpp

```diff
@@ -60,6 +60,15 @@ void Robot::RobotInit()
     m_datalogger = DragonDataLoggerMgr::GetInstance();
 
     auto path = AutonUtils::GetPathFromTrajectory("BlueLeftInside_I"); // load choreo library so we don't get loop overruns during autonperiodic
+
+    if (m_dragonswerveposeestimator != nullptr)
+    {
+        auto visionPoseEstitmators = m_dragonswerveposeestimator->GetVisionPoseEstimators();
+        if (visionPoseEstitmators.size() > 0)
```

Comment:
Do we want to assume the position or should we loop over the vision pose estimators and cast to dragonquest and when it isn't nullptr use it and break?

May even want to create a convenience method that does this camera config mgr